src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    public enum Color {
+        RED,
+        BLUE,
+        GREEN,
+        ORANGE;
+    }
```

Comment:
Color enum shouldnt' be nested class
```suggestion
    public enum Color {
        RED,
        BLUE,
        GREEN,
        ORANGE;
    }
```